sdk/src/beta9/abstractions/base/runner.py

```diff
@@ -272,6 +272,14 @@ def _sync_content(
         except Exception as e:
             terminal.warn(str(e))
 
+    def _parse_gpu(self, gpu: Union[GpuTypeAlias, List[GpuTypeAlias]]) -> str:
```

Comment:
does this handle type assertions / validation on the input?

Reply:
yes